scripts/Muon/MaxentTools/multimaxalpha.py

```diff
@@ -56,9 +56,15 @@ def MULTIMAX(
                                                               MISSCHANNELS_mm, MAXPAGE_f, PULSESHAPE_convol, DETECT_e, SAVETIME_i2, mylog)
         # outout per-iteration debug info
         if(phaseconvWS):
-            for k in range(POINTS_ngroups):
-                phaseconvWS.dataX(k)[j + 1] = (j + 1) * 1.0
-                phaseconvWS.dataY(k)[j + 1] = SENSE_phi[k]
+            offset = 0
+            for k in range(POINTS_ngroups+len(deadDetectors)):
+                if k+1 in deadDetectors:
+                    offset+=1
+                    phaseconvWS.dataX(k)[j + 1] = (j + 1) * 1.0
```

Comment:
`dataX` calls in Python can be quite costly. I would create a variable to reference the kth X row each time around the loop:
```python
phaseConvX = phaseconvWS.dataX(k)
...
phaseConvX[j+1] = (j + 1) * 1.0
```

```